packages/fxa-content-server/app/scripts/templates/sign_in_totp_code.mustache

```diff
@@ -28,9 +28,12 @@
 
     </form>
 
+    {{^hideTotpAlternatives}}
     <div class="links centered">
       <a id="use-recovery-code-link" href="/signin_recovery_code">{{#t}}Use recovery code{{/t}}</a>
       <a class="different-account-link" href="/signin">{{#t}}Use a different account{{/t}}</a>
     </div>
+    {{/hideTotpAlternatives}}
```

Comment:
Do we need another followup bug to allow the use of recovery codes in the pairing flow?

Reply:
in https://github.com/mozilla/fxa/issues/1297